syz-manager/manager.go

```diff
@@ -584,10 +586,17 @@ func (mgr *Manager) runInstance(index int) (*Crash, error) {
 		log.Logf(0, "vm-%v: running for %v, restarting", index, time.Since(start))
 		return nil, nil
 	}
+
+	instanceName := fmt.Sprintf("vm-%d", index)
+	var machineInfo []byte
```

Comment:
It seems reasonable to indicate "no machine info" by returning nil, then this can be just:
```
machineInfo := mgr.serv.getMachineInfo(instanceName)
```